### ISSUE TITLE
BUG only shows 1 Remote 

### ISSUE THREAD

**Issue by barzoock** [prasad-edlabadka/homebridge-tuya-ir#58]
Since last update (13 Jul 22׳) It shows only 1 remote (I had a few configed - and it worked great).
I re-confied everything and forced restart all hb services and clean all cache - it didn't fix.
Also added manually all remote IDs. 
Please help me. Thank you

**Comment by prasad-edlabadka** [comment]
Can you please send logs to debug? Also please ensure you have v1.0.2. Version v1.0.1 had a bug that I just fixed

**Comment by barzoock** [comment]
How can i send you logs without showing my device ID ?

**Comment by alszaitsev** [comment]
Can confirm. The same error - only one AC remote added.

`
[7/13/2022, 8:38:32 PM] [TuyaIR] {"result":{"access_token":"1****","expire_time":2793,"refresh_token":"****","uid":"****"},"success":true,"t":****,"tid":"****"}
[7/13/2022, 8:38:32 PM] [TuyaIR] Login successful.
[7/13/2022, 8:38:32 PM] [TuyaIR] Fetching configured remotes...
[7/13/2022, 8:38:32 PM] [TuyaIR] Auto discovery of remotes enabled. Fetching with API...
[7/13/2022, 8:38:32 PM] [homebridge-weather-plus] This plugin generated a warning from the characteristic 'Air Pressure': Characteristic not in required or optional characteristic section for service TemperatureSensor. Adding anyway.. See https://homebridge.io/w/JtMGR for more info.
[7/13/2022, 8:38:32 PM] [TuyaIR] API returned 1 remotes...
[7/13/2022, 8:38:32 PM] [TuyaIR] https://openapi.tuyaeu.com/v1.0/devices/bfb40d1f5ebcae38d0a1qm
[7/13/2022, 8:38:32 PM] [TuyaIR] Adding new accessory: Air Conditioning AZ
[7/13/2022, 8:38:32 PM] [TuyaIR] Removing 1 accessories as they are no longer configured...
[7/13/2022, 8:38:32 PM] [TuyaIR] Trying to login...
[7/13/2022, 8:38:32 PM] [TuyaIR] Logging in to the the server https://openapi.tuyaeu.com...
[7/13/2022, 8:38:32 PM] [TuyaIR] {"url":"https://openapi.tuyaeu.com/v1.0/token?grant_type=1","headers":{"client_id":"***","sign":"*****","t":*****,"sign_method":"HMAC-SHA256","nonce":""}}
[7/13/2022, 8:38:32 PM] [TuyaIR] {"result":{"access_token":"*****","expire_time":2792,"refresh_token":"*****","uid":"*****"},"success":true,"t":****,"tid":"*****"}
[7/13/2022, 8:38:32 PM] [TuyaIR] Login successful.
[7/13/2022, 8:38:32 PM] [TuyaIR] Fetching configured remotes...
[7/13/2022, 8:38:32 PM] [TuyaIR] Auto discovery of remotes enabled. Fetching with API...
[7/13/2022, 8:38:33 PM] [TuyaIR] API returned 1 remotes...
[7/13/2022, 8:38:33 PM] [TuyaIR] https://openapi.tuyaeu.com/v1.0/devices/bf4cac522270fcf42d8u4q
[7/13/2022, 8:38:33 PM] [TuyaIR] Restoring existing accessory from cache: Air Conditioning Space
[7/13/2022, 8:38:33 PM] [TuyaIR] Removing 0 accessories as they are no longer configured...`

So basically I have 2 remotes - the first returns Air Conditioning AZ and then clears it

**Comment by alszaitsev** [comment]
restoring to 1.0.1 doesn't work

**Comment by alszaitsev** [comment]
Changing the plugin mode to child bridge solves the problem

**Comment by prasad-edlabadka** [comment]
Are you using multiple Smart IR devices?

**Comment by alszaitsev** [comment]
When restarting - it adds sometimes the ACs but only one is shown on homekit and it is not working with access denied error. After second restart al деру devices ale deleted. New restart again adds them all but still only one is visible in Apple Home and it doesn't react (works only on the side of homebridge)

**Comment by alszaitsev** [comment]
The ACs appear in homebridge after several restarts, but they are not appearing AppleHomekit (only one of ACs is visible but with no answer error)

**Comment by prasad-edlabadka** [comment]
I found the bug. Fixing it. Should be release in an hour.

**Comment by prasad-edlabadka** [comment]
Fixed in release v1.0.4. Please give it a go.

**Comment by barzoock** [comment]
> Fixed in release v1.0.4. Please give it a go.

Accessories are now shown but controlling them does nothing...

**Comment by barzoock** [comment]
> > Fixed in release v1.0.4. Please give it a go.
> 
> Accessories are now shown but controlling them does nothing...

Well, after restarting several times it works.... hope it will stay like this

**Comment by prasad-edlabadka** [comment]
There are fixes for DIY in v1.0.5 which should stablize problems. Please test and let me know.
